config/kinoite-packages.yml

```diff
@@ -1,18 +1,12 @@
 type: rpm-ostree
 repos:
   # - https://blah
-install:    
+install:
 remove:
-  - kmousetool 
+  - kmousetool
   - plasma-welcome
-  - kfind
-  - kcharselect
-  - kde-connect
-  - kde-connect-libs
-  - kdeconnectd
```

Comment:
please also ensure that kdeconnectd is not running by default when included. If it is, add config to disable it by default

Reply:
could I put a systemctl disable script in config/scripts?